### PR TITLE
Adding New Contributor Ambassador role to K8s

### PR DESCRIPTION
**Pull request by jaredbhatti**
Adding new contrib ambassador role to k8s with details of the role and responsibilities.

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/en/docs/contribute/advanced.md [hunk 1]
```diff
@@ -110,6 +110,25 @@ The SIG Docs representative for a given release coordinates the following tasks:
...
+
+Responsibilities for New Contributor Ambassadors include: 
+
+- Being available on the [Kubernetes #sig-docs channel](https://kubernetes.slack.com) to answer questions from new contributors.
+- Working with PR wranglers to identify good first issues for new contributors. 
+- Mentoring new contributors through their first few PRs to the docs repo. 
+- Helping new contributors create the more complex PRs they need to become Kubernetes members.
+- [Sponsoring contributors](/docs/contribute/advanced/#sponsor-a-new-contributor) on their path to becoming Kubernetes members. 
+
+Current New Contributor Ambassadors are announced at each SIG-Docs meeting, and in the [Kubernetes #sig-docs channel](https://kubernetes.slack.com).
```
Comment (Bradamant3) [inline content/en/docs/contribute/advanced.md]: And we can add about the wiki page after it's a thing?

Comment (jaredbhatti) [reply content/en/docs/contribute/advanced.md]: Yep!

**Pull request merged**